source/application/CLS/CLSAppController.cpp

```diff
@@ -1,13 +1,21 @@
 #include "CLSAppController.h"
 
 #include "dataman/AMRun.h"
-
+#include "dataman/CLS/CLSDbUpgrade1Pt1.h"
 #include "beamline/CLS/CLSStorageRing.h"
 
 CLSAppController::CLSAppController(CLSAppController::CLSBeamlineID facilityId, QObject *parent) :
     AMAppController(parent)
 {
 	facilityId_ = facilityId;
+
+	// Append the AM upgrade 1.7 to the list for the user database
+	AMDbUpgrade *am1Pt7UserDb = new CLSDbUpgrade1Pt1(facilityId_, "user", this);
```

Comment:
It's a little confusing for the CLSDbUpgrade1Pt1 class to be instantiated with a variable named am1Pt7UserDb. Should they be similar?